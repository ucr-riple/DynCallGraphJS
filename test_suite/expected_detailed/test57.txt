{
    'global (test57.js:1:1:10:5)': [
        'anon (test57.js:1:2:10:2)'
    ],
    'anon (test57.js:8:11:8:38)': [
        'apply (Native)(test57.js:8:11:8:38)'
    ],
    'apply (Native) (test57.js:8:11:8:38)': [
        'anon (test57.js:2:17:4:4)'
    ],
    'anon (test57.js:9:3:9:6)': [
        'anon (test57.js:5:14:7:4)'
    ]
}