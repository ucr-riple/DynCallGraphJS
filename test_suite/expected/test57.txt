{ '(test57.js:1:1:10:5)':
   [ '(test57.js:1:2:10:2)' ],
  '(test57.js:8:11:8:38)': [ 'apply (Native)' ],
  'apply (Native)':
   [ '(test57.js:2:17:4:4)' ],
  '(test57.js:9:3:9:6)':
   [ '(test57.js:5:14:7:4)' ] }